src/beast/beast/nudb/detail/gentex.h

```diff
@@ -73,7 +73,7 @@ gentex_t<_>::lock()
 
 template <class _>
 void
-gentex_t<_>::unlock()
+gentex_t<_>::finish()
 {
     std::unique_lock<
```

Comment:
By splitting the declaration of the local variable we bring the ratio between the longest and second longest lines down to 1.125 from 1.444. The code is formatted using subjective rules to create eye-pleasing shapes which optimize the properties of foveal vision. Here's a side by side comparison of foveal optimized versus traditional formatting, with visual features highlighted in red:
http://i.imgur.com/qhIBL81.png